README.md

```diff
@@ -101,6 +101,14 @@ Make sure that you install somewhere in your `$PATH`.
     
     NOTE: You might need to wait for a few minutes for the namespace to be fully cleaned up.
 
+### Schedule
```

Comment:
Maybe add some text saying that the step that created the backup was a one-time thing, and if you want to have Ark take backups at regularly scheduled intervals, you can do it like this. WDYT?